source/fundamentals/atlas-search.txt

```diff
@@ -683,3 +676,89 @@ The search returns the following document:
 
 To learn more about the ``wildcard`` operator, see the :atlas:`wildcard </atlas-search/wildcard>` 
 Atlas guide.
+
+Modify Atlas Search Behavior
+----------------------------
+
+You can modify the behavior of the ``Search()`` method by passing
+a ``SearchOptions`` object as parameter.
+
+The ``SearchOptions`` class contains the following properties:
+
+.. list-table::
```

Comment:
S: Include data type and default value (if any). [Example here](https://github.com/mongodb/docs-csharp/pull/517/files#diff-2d3df197496b309c7ddd45a97d9308d4773d158612af33c32040d5c8296f641eR37)